policy/modules/system/systemd.te

```diff
@@ -202,6 +203,10 @@ init_system_domain(systemd_update_done_t, systemd_update_done_exec_t)
 type systemd_update_run_t;
 files_type(systemd_update_run_t)
 
+type systemd_user_runtime_t;
+files_type(systemd_user_runtime_t)
```

Comment:
Remember that systemd-logind wants to delete all of /run/user/UID when a user logs out. So you may want to create a userdom_user_runtime_content_type, so that you can use that to allow systemd-logind to delete all userdom_user_runtime_content_type

Reply:
I would be surprise if refpolicy has not yet implemented this?

Reply:
https://github.com/SELinuxProject/refpolicy/blob/master/policy/modules/system/userdomain.te#L78

so you want something like: userdom_user_runtime_content(system_user_runtime_t)
so that systemd-logind can delete it when you log out